premium/usage.go

```diff
@@ -410,6 +410,9 @@ func (u *BatchUpdater) CheckQuota(ctx context.Context) (CheckQuotaResult, error)
 		return CheckQuotaResult{HasQuota: false}, fmt.Errorf("failed to get usage: %w", err)
 	}
 	if usage.StatusCode() != http.StatusOK {
+		if u.tokenClient.GetTokenType() == auth.APIKey && usage.StatusCode() == http.StatusForbidden {
+			u.logger.Warn().Msg("API Key may have expired. Please see the CloudQuery Console see the expiration status.")
```

Comment:
I think you could also get a 403 if it's invalid, so maybe we can say 

```
API Key invalid or may have expired. Please see the CloudQuery Console for details.
```

Better would be to return a specific message from the API if it has expired, and then telling the user so, but this would already be an improvement